nix_update/__init__.py

```diff
@@ -26,6 +26,12 @@ def parse_args() -> Options:
     parser.add_argument(
         "--commit", action="store_true", help="Commit the updated package"
     )
+    parser.add_argument(
+        "-ve",
```

Comment:
Does argparse allow single character flags?

Reply:
yep, that's how I have been using it